### PR TITLE
aws-nuke: init at 2.14.0

### PR DESCRIPTION
**Pull request by grahamc**
###### Motivation for this change

Package https://github.com/rebuy-de/aws-nuke

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -967,6 +967,8 @@ in
 
   aws-google-auth = python3Packages.callPackage ../tools/admin/aws-google-auth { };
 
+  aws-nuke= python3Packages.callPackage ../tools/admin/aws-nuke { };
```
Comment (JohnAZoidberg) [inline pkgs/top-level/all-packages.nix:970]:
```suggestion
  aws-nuke = python3Packages.callPackage ../tools/admin/aws-nuke { };
```

Comment (JohnAZoidberg) [reply pkgs/top-level/all-packages.nix:970]: Why python3Packages when it's a go project?

Comment (grahamc) [reply pkgs/top-level/all-packages.nix]: Fixed, thanks, good catch!

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -967,6 +967,8 @@ in
 
   aws-google-auth = python3Packages.callPackage ../tools/admin/aws-google-auth { };
 
+  aws-nuke = python3Packages.callPackage ../tools/admin/aws-nuke { };
```
Comment (NeQuissimus) [inline pkgs/top-level/all-packages.nix:970]: Should be the regular `callPackage`, no?

Comment (grahamc) [reply pkgs/top-level/all-packages.nix:970]: Hah. My copy-paste is leaking!

Comment (grahamc) [reply pkgs/top-level/all-packages.nix:970]: Fixed :sweat_smile:

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 108420` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>aws-nuke</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 108420` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>aws-nuke</li>
</ul>
</details>

**Pull request merged**